src/headers/tfm.h

```diff
@@ -318,8 +318,8 @@
 
    typedef ulong64            fp_digit;
 #define SIZEOF_FP_DIGIT 8
+#define DIGIT_SHIFT     6
```

Comment:
This could go into `tfm_private.h` or even `fp_prime_random_ex.c` since it's used nowhere else and shouldn't be public!? or am I missing something?

Reply:
I dunno.  It is, after all, useful if you do want to handle bits in a bignum, and considering how public fp_digit and fp_int are anyway...

Reply:
> considering how public fp_digit and fp_int are anyway...

yeah ...

Would it then maybe make sense to make `fp_[on|off]_bitnum()` public? 

ah nvm, I'll merge it as is, if someone needs it, they can still make it public or so.